v2/client/handlers.go

```diff
@@ -0,0 +1,170 @@
+package client
+
```

Comment:
Please can we document why we need handlers for V2 & V1 STOP streams ?

Reply:
It's obviously for backwards compatibility and drop in replacement, but I will add some documentation in the client struct; it doesn't belong here.